controllers/ProposalController.php

```diff
@@ -570,4 +572,91 @@ private function removeExistingUploadedFile(string $filepath)
             throw new CannotDeleteFileException();
         }
     }
+
+
+    public function actionPostComment()
+    {
+        $model = new ManageCommentForm();
+
+        if ($model->load(Yii::$app->request->post()) && $model->validate()) {
+            $referrer = explode('/', Yii::$app->request->referrer);
+            $proposalId = $referrer[count($referrer) - 1];
+            $this->saveComment($model->content, $proposalId);
+        }
+
+        return $this->redirect('/proposal/my-proposals/'. $proposalId);
+    }
+
+    public function actionEditComment()
+    {
+        $model = new ManageCommentForm();
+        $model->id = Yii::$app->request->post()['ManageCommentForm']['id'];
+        if ($model->load(Yii::$app->request->post()) && $model->validate()) {
+            $supposedCommentId = $model->id;
+            $originalComment = $this->checkIfCommentExists($supposedCommentId);
+            $this->checkIfUserIsOwnerOfComment($originalComment);
+            $this->checkIfCommentIsFromCurrentProposal($originalComment);
+            $this->saveEditedComment($originalComment, $model->content);
+        }
+
+        return $this->redirect('/proposal/my-proposals/' . $originalComment->proposal_id);
+    }
+
+    private function saveComment(string $commentInput, int $proposalId)
+    {
+        $comment = new Comment();
+        $comment->proposal_id = $proposalId;
+        $comment->content = $commentInput;
+        $comment->author_id = MainController::getCurrentUser()->id;
+        $comment->date = Util::getDateTimeFormattedForDatabase(new \DateTime());
+        $comment->edited_date = $comment->date;
```

Comment:
Why not null ? I think it's better for a non-edited comment to have a null edited_date